x-pack/docs/en/security/authentication/remote-clusters-privileges.asciidoc

```diff
@@ -0,0 +1,296 @@
+[[remote-clusters-privileges]]
+=== Configure remote cluster privileges
+After <<remote-clusters-connect,connecting remote clusters>>, you create a
+user role on both the local and remote clusters and assign necessary privileges.
+These roles are required to use {ccr} and {ccs}.
+
+IMPORTANT: You must use the same role names on both the local
+and remote clusters. For example, the following configuration for {ccr} uses the
+`remote-replication` role name on both the local and remote clusters. However,
+you can specify different role definitions on each cluster.
+
+You can manage users and roles from Stack Management in {kib} by selecting
+*Security > Roles* from the side navigation. You can also use the
+<<security-role-mapping-apis,role management APIs>> to add, update, remove, and
+retrieve roles dynamically. When you use the APIs to manage roles in the
+`native` realm, the roles are stored in an internal {es} index.
+
+The following requests use the
+<<security-api-put-role,create or update roles API>>. You must have at least the
+`manage_security` cluster privilege to use this API. 
+
+[[remote-clusters-privileges-ccr]]
+//tag::configure-ccr-privileges[]
+==== Configure privileges for {ccr}
+The {ccr} user requires different cluster and index privileges on the remote
+cluster and local cluster. The following requests create separate roles on the
+local and remote clusters, and then create a user with the required roles.
+
+[discrete]
+===== Remote cluster
+On the remote cluster that contains the leader index, the {ccr} role requires
+the `read_ccr` cluster privilege, and `monitor` and `read` privileges on the
+leader index.
+
+NOTE: If requests will be issued <<run-as-privilege,on behalf of other users>>,
+then the the authenticating user must have the `run_as` privilege on the remote 
+cluster.
+
+The following request creates a `remote-replication` role on the remote cluster:
+
+[source,console]
+----
+POST /_security/role/remote-replication
+{
+  "cluster": [
+    "read_ccr"
+  ],
+  "indices": [
+    {
+      "names": [
+        "leader-index-name"
+      ],
+      "privileges": [
+        "monitor",
+        "read"
+      ]
+    }
+  ]
+}
+----
+
+////
+[source,console]
+----
+DELETE /_security/role/remote-replication
+----
+// TEST[continued]
+////
+
+[discrete]
+===== Local cluster
+On the local cluster that contains the follower index, the `remote-replication`
+role requires the `manage_ccr` cluster privilege, and `monitor`, `read`, `write`,
+and `manage_follow_index` privileges on the follower index.
+
+NOTE: If you are managing
+<<remote-clusters-connect,connecting to the remote cluster>> using
+the cluster update settings API, you will also need a user with the `all`
+cluster privilege.
```

Comment:
I think we can drop this note now since there is a dedicated page for "connecting to the remote cluster". This now feels out of place.

Reply:
Agreed - I'll remove that note.